### DISCUSSION TITLE
[feature request] useIsMutation API support blacklist

### DISCUSSION THREAD

**Discussion by Codyooo** [TanStack/query#4209] [Q&A] [answered]
I want to show loading status for almost every request. But there are few certain request that I don't want show it's loading status on the UI, like the saveDraft request, because it's fetching in the background and it is frequently called.
I wonder is there a way to exclude certain request when using the useIsMutating API。

`
  const mutatingCount = useIsMutating({exclude:['saveDraft']});
`

**Comment by TkDodo** [comment, marked as answer]
there is no such thing, except that you can pass a `predicate` function that will be called with every key and you can return true / false to include it. This goes for [all filters](https://tanstack.com/query/v4/docs/guides/filters)
